graphql-api/src/lodash.d.ts

```diff
@@ -0,0 +1,3 @@
+// There's a @types package for lodash but adding it as a dependency
```

Comment:
Strange

Reply:
could very well be, will try

Reply:
@mattsolo1 ...except that even though it passes CI, now trying to build an image for deployment crashes. Putting this back like it was.